pastis/matrix_generation/matrix_from_efields.py

```diff
@@ -135,6 +142,40 @@ def setup_single_mode_function(self):
                                                     self.luvoir, self.resDir, self.save_efields, self.saveopds)
 
 
+class MatrixEfieldRST(PastisMatrixEfields):
+    """
+    Class to calculate RST Electrical field of RST CGI.
```

Comment:
Just found this - this class calculates the PASTIS matrix from E-fields, not just the E-fields themselves.